### ISSUE TITLE
about settings

### ISSUE THREAD

**Issue by YeewZ** [Meihan-Liu/24AAAI-A2GNN#2]
Hello, I would like to inquire about the specific hardware parameters. When I replicated your code on the RTX 3090, there was an average discrepancy of two points in the results. Should I adjust the learning rate (lr) in the MMD or make some other modifications?"

**Comment by Meihan-Liu** [comment]
Our experiments are performed on NVIDIA Tesla V100 card with 32G. The python libraries we use to implement our experiments are Python 3.8, Pytorch 1.13.1, Pytorch-geometric 2.4.0, Pytorch-sparse 0.6.15 and Pytorch-scatter 2.1.0.
Hyperparameters can refer to url(https://github.com/pygda-team/pygda/blob/main/benchmark/run_citation.sh). You can search learning rate or weight decay in the range [0.0001, 0.01].

**Comment by YeewZ** [comment]
Thank you very much for providing answers to my questions.

**Comment by sepideh-nsf** [comment]
Hi,
Thank you for the information provided and the interesting work!
I am looking for the exact setup you have used for running experiments with different combinations of datasets as the source and target domain. Unfortunately, the link you provided here does not work. I'd appreciate it if you give some details on the setups.

**Comment by lmh760008522** [comment]
Recently, we have added more experiments and refreshed the folder. The new link is here: https://github.com/pygda-team/pygda/blob/main/benchmark/node/run_citation.sh

**Comment by sepideh-nsf** [comment]
Thank you so much for your response!
Also, could you please provide a link to the Supplementary Material? I couldn't find it. Thanks!

**Comment by Meihan-Liu** [comment]
You can find appendix in https://arxiv.org/abs/2407.11052 !

**Comment by sepideh-nsf** [comment]
I appreciate it!
